docs/components/list-group.md

```diff
@@ -31,16 +31,16 @@ Add labels to any list group item to show unread counts, activity, etc.
 {% example html %}
 <ul class="list-group">
   <li class="list-group-item">
-    <span class="label label-default label-pill pull-xs-right">14</span>
     Cras justo odio
+    <span class="label label-default label-pill pull-right">14</span>
```

Comment:
There's no `pull-right` class anymore. Stick with `pull-xs-right`
Also, won't putting the floats last instead of first cause some breakage when flexbox is disabled?